schema/device.go

```diff
@@ -12,6 +12,13 @@ const (
 	DeviceModelRubix        = "Rubix"
 )
 
+type DeviceKey struct {
+	Type  string `json:"type" default:"string"`
+	Title string `json:"title" default:"Device Key"`
+	Min   int    `json:"minLength" default:"32"`
```

Comment:
@maharjanaman I believe these have to be `maximum` and `minimum`. please confirm

Reply:
For JSON, with `minLength` it's working with the UI (RCE). Are you referring to `struct` property `Min`. Have left similar comment in another comment too.
In JSON, we have `minLength` for string type and `minimum` for number type.